_config.yml

```diff
@@ -16,7 +16,7 @@ gems: [jekyll-sitemap]
 # you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
-title: "Insider's Guide to LDF16"
+title: "Inside Guide to LDF16"
```

Comment:
@thomdorkings You could rename this in the manifest as well if you want. You could just review every copy there. Just the short name needs to be quite short.